### ISSUE TITLE
gMap production of count maps is broken

### ISSUE THREAD

**Issue by cmillion** [cmillion/gPhoton#263]
They come out blank. I seem to have broken it when I was fixing intensity maps. But intensity maps still work!

**Comment by dtn5ah** [comment]
Do the FITS Tables come out as NaNs or as a constant value? (In reference for #267)

**Comment by cmillion** [comment]
A constant. I think the constant was zero, but any constant across the image will make a "blank" image.

**Comment by scfleming** [comment]
#267 comes out all a constant value (1656 if I recall).  Run on my Mac, the same command doesn't have this issue.

**Comment by cmillion** [comment]
I fixed count map creation. Intensity maps are temporarily of lower photometric quality until we get the pre-computed deadtime table up. I also made a bunch of tests for image creation. They pass for Python 3.x but fail in 2.x with `AttributeError: 'bool' object has no attribute 'any'` on line 476 of imagetools.py, which is a bit of logic that I added to make sure that "no data" visits always return `None`. I'm unable to reproduce this on my own machine, so it might be a non-issue. But I'm going to figure out a way to avoid using `any()`, I guess, to dodge it.
